src/components/RichTextEditor/Plugins/MentionPlugin/MentionMarkupElement/MarkupElementNode.tsx

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable react/display-name */
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React from 'react';
+import { IconPeople, IconSize } from '@foundation/Icon';
+import { MentionableCategory } from '../types';
+import { MarkupElementNodeComponentType, MarkupElementNodeType, MarkupElementProps } from './types';
+
+const MarkupElementNode = ({ attributes, element, nodeProps, children }: MarkupElementProps) => (
+    <span
+        {...attributes}
+        data-slate-value={element.value}
+        data-slate-key={element.key}
+        data-slate-category={element.category}
+        contentEditable={false}
+        data-cy={`mention-${element.value?.replaceAll(' ', '-')}`}
```

Comment:
`data-cy` looks like to be a cypress tests selector but you don't use it

Reply:
Removed